src/index.html

```diff
@@ -0,0 +1,1366 @@
+<!DOCTYPE html>
```

Comment:
this seems like a mistake? like you accidentally added the index file from your webpack branch

Reply:
I'm getting incorrect filter names now from using html on development branch and applying same changes:

![image](https://user-images.githubusercontent.com/3753457/51202345-67ba8380-18b3-11e9-9b2a-3c4a91845497.png)